tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""Pytest conftest module containing common test configuration and fixtures."""
+import shutil
+
+import pytest
+from sphinx.testing.path import path
+
+pytest_plugins = "sphinx.testing.fixtures"
+
+
+def copy_srcdir_to_tmpdir(srcdir, tmp):
+    srcdir = path(__file__).parent.abspath() / srcdir
+    tmproot = tmp / path(srcdir).basename()
+    shutil.copytree(srcdir, tmproot)
+    return tmproot
+
+
+def clean_up_tmpdir(targetdir):
+    shutil.rmtree(targetdir, True)
+
+
+@pytest.fixture()
+def create_app(make_app, sphinx_test_tempdir, srcdir):
+    # copy plantuml.jar to current test temdir
+    plantuml_jar_file = path(__file__).parent.abspath() / "doc_test/utils"
```

Comment:
Do we need to change to copy it everytime for each test?

Maybe symbolic link would do the same on linux.

Or just set the right, relative path in the conf.py files of each test_docs-project